Utilities/convertFromDepsCloud.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs')
+      path = require('path')
+      url = require('url')
+
+const depsInputPath =
+    process.env.MARINER_DEPSCLOUD_INPUT ||
+    path.join(__dirname, '..', 'InputFiles', 'candidate.json');
+const depsOutputPath =
+    process.env.MARINER_DEPSCLOUD_OUTPUT ||
+    path.join(__dirname, '..', 'OutputFiles', 'candidate-converted.json');
+
+function convert(deps) {
+  result = {};
+  for(var entry in deps) {
+    const fullPath = url.parse(deps[entry].repository_url).pathname;
```

Comment:
You're also going to need to get the hostname to filter out non-github URLs.

Reply:
I understand from #11 that we're going to be working with GitHub URLs, but also supporting others, including git URLs.

From the issue, some examples:
```
"repository_url": "http://svn.apache.org/something"
"repository_url": "https://github.com/OWNER/REPO/something"
"repository_url": "https://github.com/OWNER/REPO/something/something"
"repository_url": "scm:git:git://github.com/OWNER/REPO.git/something"
```

Reply:
Correct. If the Repo URL is a git URL, we should be able to get OWNER/REPO out of it.

Reply:
I was testing this against a large internal candidates.json file (that I can't share, unfortunately), and I encountered another repository_url format that I'd missed before.

`"repository_url": "http://some.otherdomain.com/"`

This throws an error on line 21, because repo is undefined. I see a few like this in the large test file. I think that may be what Mya is referring to.

I'll amend the issue to include this test case. I'm looking for additional test cases that I may have missed.

Reply:
I realize now I wasn't entirely clear in the #11 - I'd expect the first version of this utility to throw away anything that can't be identified as a Github org/repo combination. We might support other things down the line but for now it's fine to just skip them.

Reply:
I have added a couple conditionals
- One to check if the standard GitHub host is present. Looking now for alternatives to includes()
- Another to check if the URL's pathname has at least two slashes (indicating that there is an owner and repo, at minimum)

Each case falls back to skipping the current entry and moving on to the next. 
@DuaneOBrien Please test and let me know how it goes :)

Reply:
Just switched out `includes()` with `indexOf()`